style.css

```diff
@@ -0,0 +1,214 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	font-family: Georgia;
+	letter-spacing: 1px;
+	background-color: #e1e5ee;
+}
+
+header {
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	padding: 15px 0;
+	background-color: #c7ccdb;
+	color: #2a324b;
+	margin: auto;
+}
+
+.banner {
+	padding: 5px;
+	width: 80%;
+	background-color: #c7ccdb;
+	color: #2a324b;
+	margin: 20px auto;
+	border-style: solid;
+	border-width: 1px;
+	border-color: #2a324b;
+	text-align: center;
+}
+.banner:hover {
+	letter-spacing: 3px;
```

Comment:
It sounds nitpicky, but try to keep spacing consistent - add a space above this selector to match the rest of the file 😄